uaclient/apt.py

```diff
@@ -38,16 +35,16 @@ def valid_apt_credentials(repo_url, username, password):
         return True   # Do not validate
     try:
         util.subp(['/usr/lib/apt/apt-helper', 'download-file',
-                   '%s://%s:%s@%s/ubuntu/pool/' % (
-                       protocol, username, password, repo_path),
+                   '%s://%s@%s/ubuntu/dists/%s/Release' % (
+                       protocol, credentials, repo_path, series),
                    '/tmp/uaclient-apt-test'],
```

Comment:
The ESM repo will have 'dists' open to everyone, which is why I thought you had gone with pool originally. Is this handled for ESM somewhere else?

Reply:
wow, invalid merge conflict resolution by  me against master. This should not have leaked into this branch when I rebased